index.html

```diff
@@ -50,6 +50,92 @@ <h2 class="iam">I'm Sotfware Developer</h2>
       </ul>
     </div>
   </section>
+
+  <section class="work-seccion">
+    <div class="work-one">
+      <h1 class="title-work-one">My Recent Works</h1>
+      <img class="image-recent-work" src="./images/Placeholder@2x.png" alt="This is a meaningful image description">
+      <h2 class="multi-post">Multi-post stories</h2>
+      <p class="text-work-one">A daily selection of privately personalized reads; no accounts or sign-ups required. has
+        been the industry's
+        standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.
+      </p>g
+      <ul class="tag-recent-work">
+        <li> CSS</li>
+        <li>HTML</li>
+        <li>Bootstrap</li>
+        <li>Ruby</li>
+      </ul>
+      <button type="button" class="button-see-project">See project</button>
+    </div>
+    <div class="container-works grid">
```

Comment:
Figma Design          |  Your design     
:-------------------------:|:-------------------------:
![Screenshot_51](https://user-images.githubusercontent.com/69638013/164956297-505b7a3d-915e-4eb3-9b26-e4ecc0d66949.png)  |  ![Screenshot_52](https://user-images.githubusercontent.com/69638013/164956290-92fbfb08-4c62-459e-a5bc-cb70819d515a.png)


For each card kindly :

- [ ] adjust the image for the card to have same width as Figma design
- [ ]  add the right border outline to the li tags 
- [ ]  add more padding to the button, remove border outline and add button interactions to your button to look like the Figma design